project/plugins.sbt

```diff
@@ -1,4 +1,4 @@
-addSbtPlugin("com.typesafe.play" % "interplay" % sys.props.get("interplay.version").getOrElse("2.0.3"))
+addSbtPlugin("com.typesafe.play" % "interplay" % sys.props.get("interplay.version").getOrElse("2.0.4"))
```

Comment:
Is this change still necessary if we remove the build for `scala213`? (I guess it doesn't harm, correct?)

Reply:
no, this is not necessary. I did it only for the sake of maintenance